dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/tasks/sql.py

```diff
@@ -77,9 +77,7 @@ def __init__(
     @property
     def sql_type(self) -> int:
         """Judgement sql type, use regexp to check which type of the sql is."""
-        pattern_select_str = (
-            "^(?!(.* |)insert |(.* |)delete |(.* |)drop |(.* |)update |(.* |)alter ).*"
-        )
+        pattern_select_str = "^(?!(.* |)insert |(.* |)delete |(.* |)drop |(.* |)update |(.* |)alter |(.* |)create ).*"
```

Comment:
you should also add test case about SQL syntac `create table` in https://github.com/apache/dolphinscheduler/blob/0c7aa4e2c5af05a1df099e1ba29cef876446f79e/dolphinscheduler-python/pydolphinscheduler/tests/tasks/test_sql.py#L28-L64